opengovsg/postmangovsg
# Release v1.9.0

## Features
Add support for Telegram as a new channel type.

- Telegram frontend and backend (#373)
- Telegram workers (#424)
- Telegram handlers (#389)

## Deploy notes
- Migrated over non-OGP users’ subscribers and bot credentials over to production database
- Set `TELEGRAM_WEBHOOK_URL` and `DEFAULT_COUNTRY` in Elastic Beanstalk
- Deployed Telegram resource on API Gateway
- Deployed production telegram-handler lambda function
- Created tables for Telegram
- Added `TELEGRAM` to `enum_campaigns_type`

## Tests
- Add a new Telegram bot token under settings
- Subscribe to bot through Telegram client and ensure that the bot is able to successfully capture your phone number
- Send a test message to the newly subscribed number
- Proceed to create and send a Telegram campaign
